util.c

```diff
@@ -4721,13 +4708,9 @@ Perl_seed(pTHX)
     * if there isn't enough entropy available.  You can compile with
     * PERL_RANDOM_DEVICE to it if you'd prefer Perl to block until there
     * is enough real entropy to fill the seed. */
-#  ifdef __amigaos4__
-#    define PERL_RANDOM_DEVICE "RANDOM:SIZE=4"
-#  else
-#    define PERL_RANDOM_DEVICE "/dev/urandom"
-#  endif
```

Comment:
I don't like this change. If we don't support AmigaOS 4 anymore (is there an official platform list somewhere?), we should rip out all `#ifdef __amigaos4__` code in this file (as well as the `amigaos4/` subdirectory, and the `hints/amigaos.sh` file, and `README.amiga`, and `pod/perlamiga.pod`, and `ext/Amiga-ARexx`, and `ext/Amiga-Exec`).

Reply:
This pretty easy to revert (and harmless). When I checked the list of supported platforms I didn't see AmigaOS in the list. Worst case scenario if we remove this, then AmigaOS would just get it's seed like all the other platforms without `/dev/urandom/`

I think we just need guidance on what platforms are officially supported or not.